metricq/interval_source.py

```diff
@@ -70,10 +70,10 @@ async def task(self):
                 logger.info("stopping IntervalSource task")
                 break
             except asyncio.TimeoutError:
-                # This is the normal case, just continue with the loop
+                # This is the normal case, just continue with the  loop
```

Comment:
Where did the space come from?

```suggestion
                # This is the normal case, just continue with the loop
```